protocol/consistencychecks.go

```diff
@@ -199,8 +195,8 @@ func (cc *ConsistencyChecks) verifyRegistration(msg *Response,
 func (cc *ConsistencyChecks) verifyKeyLookup(msg *Response,
 	uname string, key []byte) error {
 	df := msg.DirectoryResponse.(*DirectoryProof)
-	ap := df.AP
-	str := df.STR
+	ap := df.AP[0]
+	str := df.STR[0]
```

Comment:
It is validated in https://github.com/coniks-sys/coniks-go/blob/667a6e0367cf8f8fa028feb166b9137d7075e070/protocol/consistencychecks.go#L75